src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+type Props = {};
+
+export const TodoHeader: React.FC<Props> = props => {
+  const {} = props;
+
+  return (
+    <header className="todoapp__header">
+      {/* this button should have `active` class only if all todos are completed */}
+      <button
+        type="button"
+        className="todoapp__toggle-all active"
+        data-cy="ToggleAllButton"
+      />
+
+      {/* Add a todo on form submit */}
+      <form>
+        <input
+          data-cy="NewTodoField"
+          type="text"
+          className="todoapp__new-todo"
+          placeholder="What needs to be done?"
+        />
+      </form>
```

Comment:
The form should handle adding a new todo on submit, as indicated by the comment. Currently, there is no logic to handle form submission. Consider adding an `onSubmit` handler to the form to implement this functionality.